tests/integration/testdata/config-test.yaml

```
@@ -0,0 +1,8 @@
+default-secrets: "mysecrets"
+
+secrets:
+  name: "mysecrets"
+  plugin: "azure.keyvault"
+  config:
+    vault: "REPLACE-VALUE"
```

I just realized that we don't need to do any text substitution tricks anymore to change values in the porter config file. 

You can set an environment variable when running porter and then let porter resolve the value. It's much cleaner than doing the text replacement.

```suggestion
    vault: ${env.PORTER_TEST_VAULT}
```

---

nit! Thanks for pointing it out